subgraph-api/src/api-server.ts

```diff
@@ -0,0 +1,178 @@
+import { hemi, hemiSepolia } from 'hemi-viem'
+import { mainnet, sepolia } from 'viem/chains'
+import config from 'config'
+import cors from 'cors'
+import express from 'express'
```

Comment:
On react, ESLint would throw a warning because these imports are not properly sorted 🤔  It may be a mismatch of rules.

Could it also be that `bloq/node` has more rules related to CommonJs than ESM?

Reply:
The rule that fails in the FE is `import/order`, it is configured for [bloq/next here](https://github.com/bloq/eslint-config-bloq/blob/c4f7b586abc08593dc93d8f75df840f2424fc66f/next.js#L14)

Reply:
That rule should be applied. Will check.

Reply:
The rule `import/order` was only set in `bloq/next`. I'm manually adding it.